manifests/ppa.pp

```diff
@@ -5,6 +5,8 @@
   $options        = $::apt::ppa_options,
   $package_name   = $::apt::ppa_package,
   $package_manage = false,
+  $proxy_host     = undef,
+  $proxy_port     = 8080,
```

Comment:
No, I've just been doing too much of non-Ruby/Puppet as of late. I meant hash rockets.